components/net/fetch/cors_cache.rs

```diff
@@ -112,7 +112,7 @@ pub struct BasicCORSCache(Vec<CORSCacheEntry>);
 fn match_headers(cors_cache: &CORSCacheEntry, cors_req: &CacheRequestDetails) -> bool {
     cors_cache.origin == cors_req.origin &&
         cors_cache.url == cors_req.destination &&
```

Comment:
Wait, sorry. I didn't notice that `cors_req` is an instance of `CacheRequestDetails`, so the code as it is is fine.